README.md

```diff
@@ -44,6 +44,8 @@ More information about running `docker-compose` can be found [here.](https://doc
 
 [Note : Cloner clones the whole website, to restrict to a desired depth of cloning add `--max-depth` parameter]
 
+In case of an error while running `docker-compose up`, check the port 80 by running `sudo netstat -tulnp`, if they are occupied then reffer to docker documentation to change the defalt port. This is because snare will start on 0.0.0.0 with port 80. 
```

Comment:
Move `snare will start on 0.0.0.0 with port 80` under the section `Docker build instructions` as point 4
Rephrase statement about the error with default port and please don't mention netstat, I don't want to teach people how to use it in snare README. It might be a good idea to collect such cases separately, like troubleshooting, but not in readme

Reply:
Ok, ill get it done.